README.md

```diff
@@ -121,3 +138,80 @@ $ ssh -i tests/kubevirt_rsa -p 27017 kubevirt@172.30.133.9
 ```
 
 > **NOTE:** It might take a while for the VM to completely come up before SSH can be used.
+
+### Facts
+
+* Once one of the previous resources has been created, the fact modules can be tested as well as follows:
+
+```shell
+$ ansible-playbook tests/kubevirt_vm_facts.yml
+```
+
+The above command, will gather the information for the VM stated in the playbook and produce the following ouput:
+
+```json
```

Comment:
whenever API changes, this will change too, therefore I would link it to API docs instead.